ChakraCore.Debugger.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Does it make sense to put these NuGet-specific files under `Build/NuGet` like in the main ChakraCore repo:

https://github.com/Microsoft/ChakraCore/tree/master/Build/NuGet

Reply:
Can't hurt. I'll place them in an equivalent directory...

Reply:
Done